### PR TITLE
add gclient tests

### PR DESCRIPTION
**Pull request by azoyan**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -6,14 +6,27 @@ license = "MIT"
...
+nft-io = { git = "https://github.com/gear-dapps/non-fungible-token.git", tag = "0.2.5" }
+gear-lib = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.3" }
 
 [dev-dependencies]
-gtest = { git = "https://github.com/gear-tech/gear.git", rev = "d4552434" }
+gtest = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
+sp-core = { version = "6.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable" }
+gclient = { git = "https://github.com/gear-tech/gear.git", branch = "master" }
+tokio = { version = "1.21.2", features = ["full"] }
+# gear-core = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
```
Comment (shamilsan) [inline Cargo.toml]: Dead code.
```suggestion
```

File: Cargo.toml [hunk 2]
```diff
@@ -6,14 +6,27 @@ license = "MIT"
...
+] }
+nft-io = { git = "https://github.com/gear-dapps/non-fungible-token.git", tag = "0.2.5" }
+gear-lib = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.3" }
 
 [dev-dependencies]
-gtest = { git = "https://github.com/gear-tech/gear.git", rev = "d4552434" }
+gtest = { git = "https://github.com/gear-tech/gear.git", branch = "stable" }
+sp-core = { version = "6.0.0", git = "https://github.com/gear-tech/substrate.git", branch = "gear-stable" }
+gclient = { git = "https://github.com/gear-tech/gear.git", branch = "master" }
+tokio = { version = "1.21.2", features = ["full"] }
```
Comment (shamilsan) [inline Cargo.toml]: Not the latest version.
```suggestion
tokio = { version = "1.23.0", features = ["full"] }
```

File: tests/node_tests.rs [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    let mint_payload = NFTAction::Mint {
+        transaction_id,
+        token_metadata,
+    };
+
+    let gas_info = api
+        .calculate_handle_gas(None, program_id, mint_payload.encode(), 0, true, None)
+        .await?;
+
+    let (_message_id, _) = api
```
Comment (shamilsan) [inline tests/node_tests.rs]: We can omit the result if we don't use it.
```suggestion
    api
```

File: Cargo.toml [hunk 4]
```diff
@@ -6,14 +6,27 @@ license = "MIT"
 authors = ["Gear Technologies"]
 
 [dependencies]
-gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"], rev = "d4552434" }
+gstd = { git = "https://github.com/gear-tech/gear.git", features = [
+    "debug",
+], branch = "stable" }
```
Comment (shamilsan) [inline Cargo.toml]: I like it more when one feature is in one string, but it doesn't matter.
```suggestion
gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"], branch = "stable" }
```

For example, look at some other projects.

https://github.com/rust-lang/cargo/blob/master/Cargo.toml#L29
https://github.com/tokio-rs/tokio/blob/master/tokio/Cargo.toml#L117

**Review by shamilsan — APPROVED** [review]

**Review by LouiseMedova — APPROVED** [review]

**Pull request merged**